cluster/juju/layers/kubernetes-worker/layer.yaml

```diff
@@ -14,6 +14,7 @@ includes:
   - 'interface:kube-dns'
   - 'interface:kube-control'
   - 'interface:aws'
+  - 'interface:mount'
```

Comment:
Can you get interface-mount added to https://github.com/juju/layer-index? We shouldn't land this PR until it's in there.

Reply:
Done: https://github.com/juju/layer-index/pull/27